### PR TITLE
feat: Scrolling in guild member list

### PR DESCRIPTION
**Pull request by ShadowCat117**
https://github.com/Wynntils/Artemis/assets/8337467/b014a35e-25bb-4159-a6c5-f3b40af90c2b

Also reordered some variables and added missing test suits in the model

### CODE REVIEW

**Comment by kristofbolyai** [comment]
Hm, unrelated to your PR but it's strange seeing the advanced filters in that GUI :)

File: common/src/main/java/com/wynntils/models/containers/ContainerModel.java [hunk 1]
```diff
@@ -19,19 +19,27 @@
...
+    // Test suite: https://regexr.com/7jh15
     public static final Pattern ABILITY_TREE_PATTERN =
             Pattern.compile("(?:Warrior|Shaman|Mage|Assassin|Archer) Abilities");
 
     // Test suite: https://regexr.com/7b4lf
     private static final Pattern GUILD_BANK_PATTERN =
             Pattern.compile("[a-zA-Z ]+: Bank \\((?:Everyone|High Ranked)\\)");
 
+    // Test suite: https://regexr.com/7jgvf
+    private static final Pattern GUILD_MEMBER_LIST_PATTERN = Pattern.compile("[a-zA-Z ]+: Members");
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/models/containers/ContainerModel.java]: didnt we have this pattern already for adding the search bar?

Comment (ShadowCat117) [reply common/src/main/java/com/wynntils/models/containers/ContainerModel.java]: It's slightly different but also works

**Comment by ShadowCat117** [comment]
> Hm, unrelated to your PR but it's strange seeing the advanced filters in that GUI :)

I did think that maybe we should display the old basic one in certain screens. Unless we start making screen specific filters and on this for example you could search by rank or contribution amount

File: common/src/main/java/com/wynntils/models/containers/ContainerModel.java (hunk 1 above)
Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/models/containers/ContainerModel.java]: Perhaps it's best not to duplicate the patterns then? I am not sure where it is put though..

**Comment by kristofbolyai** [comment]
> > Hm, unrelated to your PR but it's strange seeing the advanced filters in that GUI :)
> 
> I did think that maybe we should display the old basic one in certain screens. Unless we start making screen specific filters and on this for example you could search by rank or contribution amount

If you add the code to support that, it could also be a config to turn them off/on, if we ever add specific filters. I tried once but it was a bit deeper rabbit hole and I didn't feel like doing it. Most likely if you refactor the feature in a sensible way it becomes simple.

File: common/src/main/java/com/wynntils/models/containers/ContainerModel.java (hunk 1 above)
Comment (ShadowCat117) [reply common/src/main/java/com/wynntils/models/containers/ContainerModel.java]: It'll either have to be `return SearchableContainerType.getContainerType(StyledText.fromComponent(screen.getTitle())) == SearchableContainerType.MEMBER_LIST;` or adding a getter for the title pattern in SearchableContainerType

**Comment by ShadowCat117** [comment]
> If you add the code to support that, it could also be a config to turn them off/on, if we ever add specific filters. I tried once but it was a bit deeper rabbit hole and I didn't feel like doing it. Most likely if you refactor the feature in a sensible way it becomes simple.

I can add it to my backlog of things I plan to work on :)

**Review by kristofbolyai — APPROVED** [review]

**Pull request merged**